packages/ai-jsx/src/lib/anthropic.tsx

```diff
@@ -175,32 +176,34 @@ export async function* AnthropicChatModel(
   }
 
   // Embed the stream "within" an <AssistantMessage>, memoizing it to ensure it's only consumed once.
-  yield (
-    <AssistantMessage>
-      {memo(
-        (async function* (): AI.RenderableStream {
-          yield AI.AppendOnlyStream;
-          let resultSoFar = '';
-          let isFirstResponse = true;
-          for await (const completion of response) {
-            let text = completion.completion;
-            if (isFirstResponse && text.length > 0) {
-              isFirstResponse = false;
-              if (text.startsWith(' ')) {
-                text = text.slice(1);
-              }
-            }
-            resultSoFar += text;
-            logger.trace({ completion }, 'Got Anthropic stream event');
-            yield text;
-          }
-
-          logger.debug({ completion: resultSoFar }, 'Anthropic completion finished');
-          return AI.AppendOnlyStream;
-        })()
-      )}
-    </AssistantMessage>
+  let accumulatedContent = '';
+  let complete = false;
+  const Stream = async function* (): AI.RenderableStream {
```

Comment:
Any reason not to do the traditional `async function* Stream(): AI.RenderableStream {` here?